cumulusci/robotframework/tests/cumulusci/communities.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Resource  cumulusci/robotframework/CumulusCI.robot
+Library   Collections
+
+Suite Setup  run keywords
+     # Note: the first community name intentionally includes a unicode
+     # character to make sure we can handle it.
+...  Insure community exists       Kōkua   kokua
+...  AND  Insure community exists  Ohana   ohana
+
+*** Keywords ***
+Insure community exists
+    [Arguments]      ${community name}  ${url prefix}
```

Comment:
Does this keyword get exposed to end users' test suits? If so, they'll want to supply the template name.

Reply:
No, that's just a helper keyword for the tests.